training/src/main/java/wolox/training/models/Book.java

```diff
@@ -40,6 +45,9 @@
     @Column(nullable = false)
     private String isbn;
 
+    @ManyToMany(mappedBy = "books")
+    private List<User> users = new LinkedList<>();
```

Comment:
As a convention is good to initialize attributes/properties inside the constructors instead of having them in their declarations